vms/platformvm/state/state.go

```diff
@@ -1501,6 +1667,33 @@ func (s *state) loadExpiry() error {
 	return nil
 }
 
+func (s *state) loadActiveSubnetOnlyValidators() error {
+	it := s.activeDB.NewIterator()
+	defer it.Release()
+
+	for it.Next() {
```

Comment:
Super nit: Inconsistent newline between `defer it.Release()` and the for loop here compared to `loadCurrentValidators()` below.